scss/components/_case-body.scss

```diff
@@ -31,3 +31,24 @@
     width: 100%;
   }
 }
+
+// Override specific to the next milestone card and next activity card
+@media (min-width: 1200px) {
+  .civicase__summary-tab-tile--big-card.col-lg-6 {
+    width: 100%;
+  }
+
+  .civicase__summary-tab-tile--small-card.col-lg-3 {
+    width: 50%;
+  }
+}
+
+@media (min-width: 1550px) {
+  .civicase__summary-tab-tile--big-card.col-lg-6 {
+    width: 50%;
+  }
+
+  .civicase__summary-tab-tile--small-card.col-lg-3 {
+    width: 25%;
+  }
+}
```

Comment:
I think you can refactor this whole new block like this:

* Change the `.civicase__summary-tab-tile--big-card.` large column class to 12.
* change the `.civicase__summary-tab-tile--small-card` large column class to 6.
* Change the whole CSS like this:

```scss
@media (max-width: 1200px) {
  .civicase__summary-tab-tile--big-card.col-lg-12 {
    width: 50%;
  }

  .civicase__summary-tab-tile--small-card.col-lg-6 {
    width: 25%;
  }
}
```